packages/cli/src/commands/installedschema.ts

```diff
@@ -36,21 +35,25 @@ export default class InstalledSchemaAppsCommand extends APICommand<typeof Instal
 	}]
 
 	async run(): Promise<void> {
-		const config: OutputItemOrListConfig<InstalledSchemaApp, InstalledSchemaAppWithLocation> = {
+		const config: OutputItemOrListConfig<InstalledSchemaApp, InstalledSchemaApp & WithNamedLocation> = {
 			primaryKeyName: 'isaId',
 			sortKeyName: 'appName',
 			listTableFieldDefinitions,
 			tableFieldDefinitions,
 		}
 
 		if (this.flags.verbose) {
-			config.listTableFieldDefinitions.splice(3, 0, 'location')
+			config.listTableFieldDefinitions = [...listTableFieldDefinitions]
+			config.listTableFieldDefinitions.splice(3, 0, 'locationId', 'location')
+			config.tableFieldDefinitions = [...tableFieldDefinitions]
 			config.tableFieldDefinitions.splice(5, 0, 'location')
```

Comment:
The reason I'm duplicating the arrays here before splicing is so we don't modify the original array which breaks unit tests. It might be cleaner to change the arrays in the util class to functions that return the array (as a fresh copy each time) but I figured this is good enough for now.